modyn/supervisor/internal/triggers/utils.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import random
+from typing import Optional, Union
+
+import pandas as pd
+import torch
+from evidently.metrics import EmbeddingsDriftMetric
+from evidently.metrics.data_drift import embedding_drift_methods
+from modyn.supervisor.internal.triggers.model_downloader import ModelDownloader
+from modyn.supervisor.internal.triggers.trigger_datasets import DataLoaderInfo, FixedKeysDataset, OnlineTriggerDataset
+from torch.utils.data import DataLoader
+
+logger = logging.getLogger(__name__)
+
+
+def get_evidently_metrics(
+    column_mapping_name: str, trigger_config: Optional[dict] = None
+) -> list[EmbeddingsDriftMetric]:
+    """Evidently metric configurations follow exactly the four DriftMethods defined in embedding_drift_methods:
```

Comment:
I think this needs an introductory sentence on what this function is doing (like instantiating an evidently metric)

Reply:
Also, if you want to do a separate PR for your TODO 5 (which i think applies here, right), we might want to open an issue and link it here

Reply:
Comment updated. Links to issues see the PR description.